src/handlebars/archive.handlebars

```diff
@@ -1,4 +1,4 @@
-{{> header title='Archive | ' }}
+{{> header title='Archive | ' style1='<link rel="stylesheet" href="http://pagination.js.org/dist/2.0.7/pagination.css">' }}
```

Comment:
I've touched on requireJS before, but only for importing and exporting individual parts of JS files into one another... what would be the improvement it brings in this situation? This highlighted code is just injecting the links into the HTML files via the Handlebars templating.

More than happy to add RequireJS if it's necessary